src/Phoenix/PhoenixClient.ts

```diff
@@ -79,9 +89,20 @@ export class PhoenixClient {
     }
   }
 
-  private getContainerHealth(delayMs: number, containerData: { forwardingId: string; dbAccountName: string }) {
-    this.getContainerStatusAsync(containerData)
-      .then((ContainerInfo) => useNotebook.getState().setContainerStatus(ContainerInfo))
+  private async getContainerHealth(delayMs: number, containerData: { forwardingId: string; dbAccountName: string }) {
+    await this.getContainerStatusAsync(containerData)
+      .then((ContainerInfo) => {
+        if (ContainerInfo) {
```

Comment:
what happens if its null? we still need to set it right?
if null case is thrown as a separate error from getContainerStatusAsync, you dont need this if condtion